lib/livebook_web/live/file_select_component.ex

```diff
@@ -147,62 +153,92 @@ defmodule LivebookWeb.FileSelectComponent do
           </div>
         <% end %>
       </div>
-      <div class="grow -m-1 p-1 overflow-y-auto tiny-scrollbar" tabindex="-1">
+      <form
+        class="min-h-full pb-4"
+        phx-change="file_validate"
```

Comment:
We have the "new directory" input and it's in this form, doesn't it trigger the change while typing?

Reply:
It might but the phx-change is a no-op. It only exists because Phoenix requires it for file uploads. Do we want to move the form around?

Reply:
We don't want to send an unnecessary event to the server on every keystroke, no?

Reply:
I think we can wrap just the file input in `<form>` and have `phx-drop-target` on the dropzone div?

Even better if we could use `phx-change` directly on the file input, but I don't know if that would work.

Reply:
Unfortunately the form has to wrap around the whole dragable area it seems. But I pushed a fix that moves the create dir stuff out. :)

Reply:
I see, sounds good!